content/HowToCreateBitbakeRecipeGitHub.md

```diff
@@ -0,0 +1,35 @@
+<img src="../files/img/2018_MistyWest_LogoCombo_FINAL_RGB.png" alt="MistyWest" width="400"/><div style="text-align: right">back to [index](../README.md)</div>
+
+# How to create a custom bitbake recipe from a public repository
+
+1. Open a terminal and navigate to the directory where you want to store the recipe. We assume that the application to add is called **DesiredApp**(adjust the name accordingly).
+2. Create a new file with the .bb extension using your preferred text editor. For this example, we'll call the file "desiredapp.bb".
+3. Start the recipe by defining the metadata. Here's an example:
+```
+SUMMARY = "Recipe for DesiredApp"
+DESCRIPTION = "Compiles DesiredApp from a public GitHub repository"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/LICENSE;md5=4d92cd373abda3937c2bc47fbc49d690"
```

Comment:
I don't believe ${COREBASE} is needed. It needs to point to the LICENSE file inside the same repository.